<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>160</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="order_button">
       <property name="text">
        <string>Заказы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="type_button">
       <property name="text">
        <string>Услуги</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="worker_button">
       <property name="text">
        <string>Работники</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="client_button">
       <property name="text">
        <string>Клиенты</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>251</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="new_order">
       <property name="text">
        <string>Новый заказ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_worker">
       <property name="text">
        <string>Добавить работника</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_type">
       <property name="text">
        <string>Новая услуга</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
